### PR TITLE
fix: Review font-size for different pages - MEED-1622 - Meeds-io/MIPs#48

### PR DESCRIPTION
**Pull request by MayTekayaa**
This change will change:
- The font-size of the titles from 20px to 24px
- The font-size of the subtitles, texts and buttons from 14px to 16px.

### CODE REVIEW

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/drawer/StakeMeedsDrawer.vue [hunk 1]
```diff
@@ -19,7 +19,7 @@
 <template>
   <deeds-drawer ref="drawer">
     <template #title>
-      <h4>{{ title }}</h4>
+      <h4 class="title font-weight-bold">{{ title }}</h4>
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/drawer/StakeMeedsDrawer.vue]: It's may be better to define generic CSS for `h4` to use a different font-size in all dapp.

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeGovernance.vue [hunk 2]
```diff
@@ -37,7 +37,7 @@
           :src="`/${parentLocation}/static/images/discord.ico`"
           alt=""
           class="me-2 img-24px">
-        <span class="text-none">{{ $t('discussOnDiscord') }}</span>
+        <span class="text-none subtitle-1">{{ $t('discussOnDiscord') }}</span>
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeGovernance.vue]: You may need to change the `.v-btn` class in generic CSS of dapp to apply it globally (without `!important`).

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue [hunk 3]
```diff
@@ -18,10 +18,10 @@
 -->
 <template>
   <v-card flat>
-    <v-card-title class="ps-0">
+    <v-card-title class="ps-0 headline font-weight-medium">
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue]: It's may be better to define generic CSS for `.v-card-title` to use a different font-size in all dapp.

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue [hunk 4]
```diff
@@ -18,10 +18,10 @@
...
       {{ $t('deeds') }}
     </v-card-title>
-    <v-card-text class="ps-0">
+    <v-card-text class="ps-0 subtitle-1">
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue]: It's may be better to define generic CSS for `.v-card-text` to use a different font-size in all dapp.

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/farming/components/ComethLiquidityPool.vue [hunk 5]
```diff
@@ -31,10 +31,10 @@
             class="mx-auto addLiquidityIcon"
             alt="Cometh Icon">
         </v-card-text>
-        <v-card-title class="justify-center pt-0 text-center text-break">
+        <v-card-title class="justify-center pt-0 text-center text-break headline font-weight-medium">
           {{ $t('rentLiquidityOnCometh') }}
         </v-card-title>
-        <v-card-subtitle class="justify-center pt-0 text-center text-break">
+        <v-card-subtitle class="justify-center pt-0 text-center text-break subtitle-1">
```
Comment (boubaker) [inline deeds-dapp-webapp/src/main/webapp/vue-app/dapp/farming/components/ComethLiquidityPool.vue]: It's may be better to define generic CSS for `.v-card-subtitle` to use a different font-size in all dapp.

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/drawer/StakeMeedsDrawer.vue (hunk 1 above)
Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/drawer/StakeMeedsDrawer.vue]: done!

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeGovernance.vue (hunk 2 above)
Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeGovernance.vue]: done!

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue (hunk 3 above)
Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue]: done!

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue (hunk 4 above)
Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/vue-app/dapp/stake/components/StakeDeeds.vue]: done!

File: deeds-dapp-webapp/src/main/webapp/vue-app/dapp/farming/components/ComethLiquidityPool.vue (hunk 5 above)
Comment (MayTekayaa) [reply deeds-dapp-webapp/src/main/webapp/vue-app/dapp/farming/components/ComethLiquidityPool.vue]: done!

**Review by boubaker — APPROVED** [review]

**Pull request merged**